zipline/assets/asset_db_migrations.py

```diff
@@ -313,3 +360,163 @@ def _downgrade_v5(op):
+        sa.Column(
+            'sid',
+            sa.Integer,
+            unique=True,
+            nullable=False,
+            primary_key=True,
+        ),
+        sa.Column('asset_name', sa.Text),
+        sa.Column('start_date', sa.Integer, default=0, nullable=False),
+        sa.Column('end_date', sa.Integer, nullable=False),
+        sa.Column('first_traded', sa.Integer),
+        sa.Column('auto_close_date', sa.Integer),
+
+        # remove foreign key to exchange
+        sa.Column('exchange', sa.Text),
+
+        # add back exchange full column
+        sa.Column('exchange_full', sa.Text),
+    )
+    op.execute(
+        """
+        insert into
+            _new_equities
+        select
+            eq.sid,
+            eq.asset_name,
+            eq.start_date,
+            eq.end_date,
+            eq.first_traded,
+            eq.auto_close_date,
+            ex.canonical_name,
+            ex.exchange
+        from
+            equities eq
+        inner join
+            exchanges ex
+        on
+            eq.exchange == ex.exchange
+        where
+            ex.country_code in ('US', '??')
+        """,
+    )
+    op.drop_table('equities')
+    op.rename_table(tmp_name, 'equities')
+
+    # rebuild all tables without a foreign key to ``exchanges``
+    alter_columns(
+        op,
+        'futures_root_symbols',
+        sa.Column(
+            'root_symbol',
+            sa.Text,
+            unique=True,
+            nullable=False,
+            primary_key=True,
+        ),
+        sa.Column('root_symbol_id', sa.Integer),
+        sa.Column('sector', sa.Text),
+        sa.Column('description', sa.Text),
+        sa.Column('exchange', sa.Text),
+    )
+    alter_columns(
+        op,
+        'futures_contracts',
+        sa.Column(
+            'sid',
+            sa.Integer,
+            unique=True,
+            nullable=False,
+            primary_key=True,
+        ),
+        sa.Column('symbol', sa.Text, unique=True, index=True),
+        sa.Column('root_symbol', sa.Text, index=True),
+        sa.Column('asset_name', sa.Text),
+        sa.Column('start_date', sa.Integer, default=0, nullable=False),
+        sa.Column('end_date', sa.Integer, nullable=False),
+        sa.Column('first_traded', sa.Integer),
+        sa.Column('exchange', sa.Text),
+        sa.Column('notice_date', sa.Integer, nullable=False),
+        sa.Column('expiration_date', sa.Integer, nullable=False),
+        sa.Column('auto_close_date', sa.Integer, nullable=False),
+        sa.Column('multiplier', sa.Float),
+        sa.Column('tick_size', sa.Float),
+    )
+
+    # drop the ``country_code`` and ``canonical_name`` columns
+    alter_columns(
+        op,
+        'exchanges',
+        sa.Column(
+            'exchange',
+            sa.Text,
+            unique=True,
+            nullable=False,
+            primary_key=True,
+        ),
+        sa.Column('timezone', sa.Text),
+        selection_string="exchange, 'US/Eastern'",
```

Comment:
Is this just unconditionally setting the 'timezone' to US/Eastern for all exchanges in the old schema?

Reply:
If so, a comment here might be helpful.